### PR TITLE
added packages/lbvs_consent/lbvs_consent.2.1.1

### PR DESCRIPTION
**Pull request by UnixJunkie**
compatible with latest dolog library

### CODE REVIEW

File: packages/lbvs_consent/lbvs_consent.2.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  ["cp" "bin/lbvs_consent_ecfp4.py" "%{bin}%/lbvs_consent_ecfp4.py"]
+  ["cp" "src/lbvs_consent_ob_maccs" "%{bin}%/lbvs_consent_ob_maccs"]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/lbvs_consent_ecfp4.py"
+             "%{bin}%/lbvs_consent_ob_maccs"]
+]
```
Comment (kit-ty-kate) [inline packages/lbvs_consent/lbvs_consent.2.1.1/opam]: This field is unnecessary since opam 2.0

Comment (UnixJunkie) [reply packages/lbvs_consent/lbvs_consent.2.1.1/opam]: Ok, I removed it.
Sorry, you probably already told me once before.
Opam lint doesn't say anything about it though.

Comment (mseri) [reply packages/lbvs_consent/lbvs_consent.2.1.1/opam]: In practice, the remove field is useful now only if you modify some other files around (e.g. shared configurations) during install and you need to revert such changes in the remove phase. This means we cannot really deprecate it or raise a warning. See: https://opam.ocaml.org/doc/Manual.html#opamfield-remove

Can you please make the change upstream as well?

**Comment by mseri** [comment]
The failures are all due to missing or wrong versions of the external dependencies

File: packages/lbvs_consent/lbvs_consent.2.1.1/opam (hunk 1 above)
Comment (UnixJunkie) [reply packages/lbvs_consent/lbvs_consent.2.1.1/opam]: ok, corrected upstream too

**Pull request merged**